requirements_all.txt

```diff
@@ -886,6 +886,9 @@ pyW215==0.6.0
 # homeassistant.components.w800rf32
 pyW800rf32==0.1
 
+# homeassistant.components.sensor.nextbus
```

Comment:
Move, `git mv`, the sensor platform under a new `nextbus` component package. That's our new way of structuring modules.
```
homeassistant/components/nextbus/__init__.py
homeassistant/components/nextbus/sensor.py
```

Just add a docstring to the init module. Run `script/gen_requirements.py` again after the move.

Reply:
Should I do the same with the test module?

Reply:
Yes, the tests should be structured the same way.